lib/ansible/modules/packaging/os/package_facts.py

```diff
@@ -139,95 +150,275 @@
           }
         }
 '''
+import json
+
+from abc import ABCMeta, abstractmethod
+
+from ansible.module_utils._text import to_native, to_text
+from ansible.module_utils.basic import AnsibleModule, get_all_subclasses
+from ansible.module_utils.six import with_metaclass
+
+
+class TestFailed(Exception):
```

Comment:
AFAICS this exception is a `RuntimeError` semantically:
```suggestion
class TestFailed(RuntimeError):
```